doc/data/messages/m/magic-value/good.py

```diff
@@ -0,0 +1,15 @@
+import random
```

Comment:
The example is good and almost made me reconsider using the check 😄

Reply:
Thanks mate!